server/src/main/kotlin/org/jetbrains/bsp/bazel/server/sync/BazelProjectMapper.kt

```diff
@@ -343,6 +344,8 @@ class BazelProjectMapper(
       sources = getSourceJarUris(targetInfo),
       dependencies = targetInfo.dependenciesList.map { it.id },
       interfaceJars = getTargetInterfaceJars(targetInfo).map { it.toUri() }.toSet(),
+      goImportPath = targetInfo.goTargetInfo?.importpath,
+      goRoot = "external/org_golang_x_net/ipv4",
```

Comment:
We want to get target's location info here, like `bazel query <label> --output=location`